Core/src/org/sleuthkit/autopsy/centralrepository/optionspanel/GlobalSettingsPanel.java

```diff
@@ -553,12 +613,16 @@ private boolean enableButtonSubComponents(Boolean enable) {
         organizationPanel.setEnabled(enable && !ingestRunning);
         organizationTextArea.setEnabled(enable && !ingestRunning);
         manageOrganizationButton.setEnabled(enable && !ingestRunning);
+        showCasesButton.setEnabled(enable && !ingestRunning);
```

Comment:
Can you also add the labels so everything looks disabled when the central repo is disabled?